core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineTidb.java

```diff
@@ -0,0 +1,25 @@
+package com.scalar.db.storage.jdbc;
+
+import com.scalar.db.common.CoreError;
+import com.scalar.db.io.DataType;
+
+/**
+ * This implements a RdbEngine for TiDB that extends MySQL one. TiDB is MySQL compatible and uses
+ * the same connection string, so special handling is needed to instantiate it, cf. {@link
+ * RdbEngineFactory#create(JdbcConfig)}
+ */
+public class RdbEngineTidb extends RdbEngineMysql {
```

Comment:
Altering the column type from BLOB to TEXT is not supported in TiDB, so I needed to add an `RdbEngine` for TiDB